StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  PaymentSheetVerticalViewController.swift
+//  StripePaymentSheet
+//
+//  Created by Yuki Tokuhiro on 5/3/24.
+//
+
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePayments
+@_spi(STP) import StripePaymentsUI
+@_spi(STP) import StripeUICore
+import UIKit
+
+class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewController {
+    var selectedPaymentOption: PaymentSheet.PaymentOption?
+    /// The type of the payment method that's currently selected in the UI, or unknown if no payment method is selected.
+    var selectedPaymentMethodType: PaymentSheet.PaymentMethodType = .stripe(.unknown)
```

Comment:
Why not nil instead of unknown? I notice it's not optional on the `FlowControllerViewController` protocol but curious.

Reply:
Great question, I agree this should be nil. I tried making it nil but the refactor was a bit too much for the PR. Will add a TODO though.